src/entities/verification_token.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Describe the entity stored in the VerificationTokens storage table
+ */
+
+import * as t from "io-ts";
+
+import { UTCISODateFromString } from "italia-ts-commons/lib/dates";
+import { FiscalCode } from "../../generated/definitions/FiscalCode";
+import { TableEntity } from "../utils/azure_storage";
+
+export const VerificationToken = t.interface({
+  ExpireAt: UTCISODateFromString,
```

Comment:
more than an event in time that may be confusing, it's better to describe a portion of time, like `validBefore` or `invalidAfter` (depends whether you include that specific time in the interval)

also we use camel case for attribute names

Reply:
Camel case -> Are you sure you want a table to have "PartitionKey, RowKey, Version" and than "myCamelCaseProperty"? All the system tables (like the Hub* tables created by the functions) use property in PascalCase.

ExpireAt -> I'm not sure I understood what you mean. Are you just suggesting to rename ExpireAt to invalidAfter/validBefore?

Reply:
re case -> can you think of a way to make it clear these types are used to table props and because of that they have a different naming convention? perhaps a comment on top of this file? another reader may have the same concert I had

expire at -> yes. the "at" of expireAt is not clear about the `=` part of the comparison, is it still valid or invalid when `t == expireAt` ? on the other hand if you define it with `before` or `after` it's clear whether it's valid for `t < expireAt` (valid before) or `t <= expireAt` (invalid after)